src/sentry/testutils/helpers/api_gateway.py

```diff
@@ -58,11 +85,18 @@ def verify_file_body(file_body, headers):
     def request_callback(request):
         assert request.body == file_body
         assert (request.headers[key] == headers[key] for key in headers)
-        return 200, {}, b"{}"
+        return 200, {}, json.dumps({"proxy": True})
 
     return request_callback
 
 
+def provision_middleware():
+    middleware = list(settings.MIDDLEWARE)
+    if "sentry.middleware.api_gateway.ApiGatewayMiddleware" not in middleware:
+        middleware = ["sentry.middleware.api_gateway.ApiGatewayMiddleware"] + middleware
```

Comment:
@AniketDas-Tekky are you planning on putting the `ApiGatewayMiddleware` middleware as the first middleware?

Reply:
if we eventually go down the VPC route, then we can move it after authentication if we treat the proxy as an internal request. But for now, I want proxied requests to be proxied ASAP